pkgs/os-specific/linux/ixgbe/default.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  kernel,
+  srcOnly,
+  kmod,
+  linuxHeaders,
```

`fetchurl`, `srcOnly` and `linuxHeaders` are unused
```suggestion
  kernel,
  kmod,
```